src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
Its equivalent in ```BufferAttribute``` is https://github.com/mrdoob/three.js/blob/dev/src/core/BufferAttribute.js#L18

Which has just been changed from array.length which it was before